tests/snuba/api/endpoints/test_organization_tags.py

```diff
@@ -63,3 +65,72 @@ def test_no_projects(self):
         response = self.client.get(url, format="json")
         assert response.status_code == 200, response.content
         assert response.data == []
+
+    @mock.patch("sentry.options.get", return_value=1.0)
+    @mock.patch("sentry.utils.snuba.query", return_value={})
+    def test_tag_caching(self, mock_snuba_query, mock_options):
+        user = self.create_user()
+        org = self.create_organization()
+        team = self.create_team(organization=org)
+        self.create_member(organization=org, user=user, teams=[team])
+        self.create_project(organization=org, teams=[team])
+        self.login_as(user=user)
+
+        url = reverse("sentry-api-0-organization-tags", kwargs={"organization_slug": org.slug})
+        response = self.client.get(url, {"use_cache": "1", "statsPeriod": "14d"}, format="json")
+        assert response.status_code == 200, response.content
+        assert mock_snuba_query.call_count == 1
+
+        response = self.client.get(url, {"use_cache": "1", "statsPeriod": "14d"}, format="json")
```

Comment:
You should also be testing the cache actually works. Which means mock a valid result form Snuba and assert the result you get is the same in both cases.